packages/frontend/src/App.svelte

```diff
@@ -34,7 +34,7 @@ onMount(() => {
 </script>
 
 <Route path="/*" isAppMounted="{isMounted}" let:meta>
-  <main class="flex flex-col w-screen h-screen overflow-hidden bg-charcoal-700">
+  <main class="flex flex-col w-screen h-screen overflow-hidden bg-[var(--pd-content-bg)]">
```

Comment:
I am wondering if we should not use `var(--pd-invert-content-bg)` as in the preferences, it is what is used however this would make the card of the form hidden :/